samples/wrapper/wrapper.cpp

```diff
@@ -282,6 +302,10 @@ void macro_substitute(string &str) {
     fprintf(stderr, "[DEBUG] replacing '%s' with '%s'\n", "$PROJECT_DIR", pd);
 #endif
 
+    string userName = aid.user_name, teamName = aid.team_name;
```

Comment:
Please follow conventions of the surrounding code. Wrapper uses `_` to separate words, not camelCase.